<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillSettings</class>
 <widget class="QDialog" name="BillSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bill settings</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QLineEdit" name="lineEditPhoneNumber">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>103</y>
     <width>220</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCompanyName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>43</y>
     <width>220</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>73</y>
     <width>220</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>137</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>77</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>45</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Company name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>183</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>183</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMessage">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>133</y>
     <width>220</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>107</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditCompanyName</tabstop>
  <tabstop>lineEditCity</tabstop>
  <tabstop>lineEditPhoneNumber</tabstop>
  <tabstop>lineEditMessage</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
